src/pybind/mgr/dashboard/ci/cephadm/start-cluster.sh

```diff
@@ -7,6 +7,7 @@ cleanup() {
     if [[ -n "$JENKINS_HOME" ]]; then
         printf "\n\nStarting cleanup...\n\n"
         kcli delete plan -y ceph || true
+        kcli delete network ceph-dashboard -y
```

Comment:
I think cleanup once shld be just fine right? no need to everytime clear the network. So something like, `cleanup=true` intially
```suggestion
    if (cleanup) {
        kcli delete network ceph-dashboard -y
        cleanup = false
       }
```

Reply:
the cleanups are executed at the end of the job run so its fine IMO.